goldstone/core/fixtures/core_initial_data.yaml

```diff
@@ -471,3 +471,122 @@
         name: service status UP
         description: 'one or more services have changed status to UP'
         search: de730aab-e0f9-40c7-81c0-4289ddb67e96
+
+  - model: core.AlertDefinition
+    pk: f053951a-2834-43e1-a5dc-a6faf36260ed
+    fields:
+        name: service status UP
+        description: 'one or more services have changed status to UP'
+        search: de730aab-e0f9-40c7-81c0-4289ddb67e96
+
+
+
+  - model: core.AlertDefinition
+    pk: adceb329-6760-4e2b-a34e-1d4a49c5b53e
+    fields:
+        name: elapsed time exceeds limit
+        description: 'a canary test has exceeded the elapsed time limit'
+        search: 836717be-a1ac-4ab7-92ed-46923abcc383
+
+  - model: core.SavedSearch
+    pk: 836717be-a1ac-4ab7-92ed-46923abcc383
+    fields:
+        name: canary elapsed time query
+        description: 'Goldstone Canary elapsed times exceeding a maximum.'
+        owner: core
+        index_prefix: internal-*
+        doc_type: canary
+        timestamp_field: '@timestamp'
```

Comment:
unfortunately, this probably needs to be `ingestion_time` instead of `@timestamp`.  Since we're doing a walk forward on the clock based on when the query runs [last end time, now), we could miss events that got caught in the pipeline.  We can still miss them with ingestion_time, but the path to ES is potentially much shorter from logstash vs the original source of the message.